docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -39,18 +39,34 @@ Strapi currently support the following databases.
 
 ::: tab yarn
 
+To quickly create a blank project from scratch, run this command:
+
 ```bash
 yarn create strapi-app my-project --quickstart
 ```
 
+Alternatively, to use one of our [starters](https://strapi.io/starters), run this command instead:
+
+```bash
+yarn create strapi-starter my-project <starter-url>
+```
+
 :::
 
 ::: tab npx
 
+To quickly create a blank project from scratch, run this command:
+
 ```bash
 npx create-strapi-app my-project --quickstart
 ```
 
+Alternatively, to use one of our [starters](https://strapi.io/starters), run this command instead:
+
+```bash
+npx create strapi-starter my-project <starter-url>
+```
+
```

Comment:
I think the npx command is ```npx create-strapi-starter my-project <starter-url>```

See README here: https://github.com/strapi/strapi-starter-gatsby-corporate for example of both yarn and npx

Reply:
That would make much more sense with npx, thanks!